src/plugins/intel_gpu/src/kernel_selector/kernels/activation/activation_kernel_opt.cpp

```diff
@@ -90,6 +90,14 @@ bool ActivationKernelOpt::Validate(const Params& p) const {
         (params.outputs[0].GetLayout() != DataLayout::bfyx && params.outputs[0].GetLayout() != DataLayout::bfzyx))
         return false;
 
+    auto input_dt = params.inputs[0].GetDType();
+    if (input_dt == Datatype::INT8 || input_dt == Datatype::INT32) {
+        for (auto act : params.activations) {
+            if (act.function == ActivationFunction::ABS)
+                return false;
+        }
+    }
```

Comment:
this code is to avoid `activation_opt` kernel for `abs` at i8 and i32, right? Could you explain why we need this code here?

Reply:
The activation opt kernel use custom vector type 'int4' but the defined ACTIVATION_FUNC_KERNEL(input) (abs(input)) returns 'uint4' which is not compatible with 'int4'. Type conversion is not possible for vector type. So ref kernel is needed for the case, activation abs i8/i32.

Reply:
Did you check the performance impact of using activation_ref instead of opt kernel? Does it take little portion in e2e network execution?

Reply:
The ratio of activation perf is 1/765549(mcs).